payload-manifests/featuregates/featureGate-CustomNoUpgrade.yaml

```diff
@@ -0,0 +1,20 @@
+{
+    "apiVersion": "config.openshift.io/v1",
+    "kind": "FeatureGate",
+    "metadata": {
+        "creationTimestamp": null,
+        "name": "cluster"
+    },
+    "spec": {
+        "featureSet": "CustomNoUpgrade"
+    },
+    "status": {
+        "featureGates": [
+            {
+                "disabled": null,
```

Comment:
Why does this not have the `DisableKubeletCloudCredentialProviders featuregate in it? It's present in the disable list for CustomNoUpgrade

Reply:
> Why does this not have the `DisableKubeletCloudCredentialProviders featuregate in it? It's present in the disable list for CustomNoUpgrade

Custom has no set list.  The list is entirely user controlled.  I could delete this featuregate instance to avoid confusion if it helps.